src/main/java/uk/gov/hmcts/reform/orgrolemapping/data/GenericArrayUserType.java

```diff
@@ -55,20 +55,13 @@ public boolean isMutable() {
     }
 
     @Override
-    public Object nullSafeGet(ResultSet resultSet, String[] names, SharedSessionContractImplementor session,
-                              Object owner)
-            throws HibernateException, SQLException {
-        if (resultSet.wasNull()) {
+    public Object nullSafeGet(ResultSet rs, int position, SharedSessionContractImplementor session, Object owner)
```

Comment:
FYI: There is a change in behaviour here with nulls.  However the only APIs that return data affected by this are the TestingSupport RefreshJobsController so this shoulc not be a concern.